gallery/src/styles/NxCard/NxCardHorizontalExample.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+export default function NxCardVerticalExample() {
+  return (
+    <div className="nx-card-container nx-card-container--column">
+      <div className="nx-card">
+        <header className="nx-card-header">
```

Comment:
If this should be a `header` then should the `.nx-card` be a `section`?  Or should it all just be divs?

Reply:
> If this should be a `header` then should the `.nx-card` be a `section`? Or should it all just be divs?

When I was writing it I hesitated over whether `<div className="nx-card-container nx-card-container--column">` should be a `<section>`. I tend to think that the cards themselves are fine as `<div>`'s

Reply:
> When I was writing it I hesitated over whether <div className="nx-card-container nx-card-container--column"> should be a <section>. I tend to think that the cards themselves are fine as <div>'s

I'm not entirely certain, but I think the outline semantics might get confused if the cards aren't sectioning content.  I think as they are now, the header of the last card may end up applying to all content after that card up until the next header or the end of some ancestor section.  Making the cards themselves `<section>`s would avoid that ambiguity.

Reply:
> > When I was writing it I hesitated over whether  should be a . I tend to think that the cards themselves are fine as 's
> 
> I'm not entirely certain, but I think the outline semantics might get confused if the cards aren't sectioning content. I think as they are now, the header of the last card may end up applying to all content after that card up until the next header or the end of some ancestor section. Making the cards themselves `<section>`s would avoid that ambiguity.

I'm not sure entirely either but I am confident that changing the `<div>`'s to `<section>`'s doesn't hurt anything so I have done so here: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/0f736b2cb618fcc1ed51598293ed5369fe894bef